static/js/admin.js

```diff
@@ -155,16 +155,19 @@ function autoGrade() {
     }
 
     if (assignment != null && question === null && studentID == null) {
+        let i = 0;
         for (let s in students) {
             params.data.sid = s;
             params.async = false;
             let res = jQuery.ajax(params);
-            $("#autogradingform").append(`${res.responseJSON.message} for ${s} </br>`)
+            setTimeout(function(s) {
+                $("#autogradingform").append(`${res.responseJSON.message} for ${s} </br>`)
+            }, 10, s);
```

Comment:
Does this imply that all autograde messages will appear no sooner than 10 seconds after the grading starts?

Reply:
10 milliseconds